share/github-backup-utils/ghe-backup-repositories

```diff
@@ -64,6 +64,8 @@ user="${host%@*}"
 hostnames=$host
 ssh_config_file_opt=
 tempdir=$(mktemp -d -t backup-utils-restore-XXXXXX)
```

Comment:
The `tempdir` needs to be created on the remote appliance too, as per https://github.com/github/backup-utils/blob/legacy/share/github-backup-utils/ghe-backup-alambic-cluster-ng#L40.

Reply:
Whoops 😊. Good catch. Fixed in https://github.com/github/backup-utils/pull/392/commits/7b3381aa4a1fb14a42a2d2ff1fab6f3d7dcedc14